modules/aws/account.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/iam"
 )
 
-// Get the Account ID for the currently logged in IAM User.
+// GetAccountId gets the Account ID for the currently logged in IAM User.
```

Comment:
What's the point of repeating the method name here? Seems 99% likely to break when the name is changed and the comment accidentally left the old way... I noticed this PR does this on most (but not all?) of the methods and it seems like it doesn't add much, but makes maintenance harder.

Reply:
https://golang.org/doc/effective_go.html#commentary
https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences
https://blog.golang.org/godoc-documenting-go-code

Reply:
I assume you're mainly referring to this:

> If every doc comment begins with the name of the item it describes, the output of godoc can usefully be run through grep. Imagine you couldn't remember the name "Compile" but were looking for the parsing function for regular expressions, so you ran the command,
>
> $ godoc regexp | grep -i parse
>
> If all the doc comments in the package began, "This function...", grep wouldn't help you remember the name. But because the package starts each doc comment with the name, you'd see something like this, which recalls the word you're looking for.
> 
> $ godoc regexp | grep parse
>    Compile parses a regular expression and returns, if successful, a Regexp
>   parsed. It simplifies safe initialization of global variables holding
>    cannot be parsed. It simplifies safe initialization of global variables

Honest question, but, have you ever actually done that sort of search? I'm sure it happens time to time, but it seems like an exceptionally rare occurrence. On the other hand, renaming methods happens all the time, and forgetting to update the comment will be confusing for everyone, including anyone who tries to search for method names using `godoc`.

Reply:
I actually think it's quite useful. Another advantage is that it gets people to write full sentences when they're commenting exported entities. That helps a lot to improve the quality o the documentation.

As for the renaming, I think most people use a tool like [gorename] to rename variables which automatically changes the comment as well so I don't see any big risks in that.

Reply:
Hmm, OK, I'm willing to try it. I've never used `gorename`. I use IntelliJ to do refactoring. Not sure if it handles comments.

Reply:
I personally use [vim-go](https://github.com/fatih/vim-go) but I'm sure that [Go Land](https://www.jetbrains.com/go/) and the IntelliJ Go plugin also use `gorename` under the hood.